### PR TITLE
Fixes #19919: Document that escaping is incoherent with the rest in Technique Editor in 6.X, that it is corrected in 7.0, and that's a breaking change

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/19919

### CODE REVIEW

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 1]
```diff
@@ -99,7 +99,23 @@ if you feel the generated policies should be modified (for instance, if you chan
 
 image::Rule_config.png[]
 
-== Variables
+[[Variables-and-values]]
+== Variables and Values
+
+Values used in Directives, Parameters and Properties have several types - Parameters and Properties are either String or JSON, while in Directives they can be free String, String validated by regular expression, or Javascript.
+
+These values are automatically escaped to be managed correctly. The escaping doubles the backslashes ( `\` -> `\\` ), and replace a quote by a backslash quote ( `"` -> `\\"`) on Linux/Unix systems, and on Windows it replaces a quote by a backtick quote ( `` " `` -> `` `" `` ), and doubles the backticks.
```
Comment (ncharles) [inline src/reference/modules/usage/pages/configuration_management.adoc:107]: i could get to have `` \`` `` to work

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 2]
```diff
@@ -99,7 +99,23 @@ if you feel the generated policies should be modified (for instance, if you chan
 
 image::Rule_config.png[]
 
-== Variables
+[[Variables-and-values]]
+== Variables and Values
+
+Values used in Directives, Parameters and Properties have several types - Parameters and Properties are either String or JSON, while in Directives they can be free String, String validated by regular expression, or Javascript.
```
Comment (amousset) [inline src/reference/modules/usage/pages/configuration_management.adoc:105]:
```suggestion
Values used in directives, parameters and properties have several types - parameters and properties are either String or JSON, while in directives they can be free String, String validated by regular expression, or Javascript.
```

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 3]
```diff
@@ -99,7 +99,23 @@ if you feel the generated policies should be modified (for instance, if you chan
...
+
+These values are automatically escaped to be managed correctly. The escaping doubles the backslashes ( `\` -> `\\` ), and replace a quote by a backslash quote ( `"` -> `\\"`) on Linux/Unix systems, and on Windows it replaces a quote by a backtick quote ( `` " `` -> `` `" `` ), and doubles the backticks.
+
+[WARNING]
+
+====
+
+Values in the Technique Editor in Rudder 6.X are *not* escaped - their content need to be escaped, based on the previous rules. Starting in Rudder 7.0, values in Technique Editor are escaped properly.
```
Comment (amousset) [inline src/reference/modules/usage/pages/configuration_management.adoc:113]:
```suggestion
Values in the Technique Editor in Rudder 6.X are *not* escaped - their content needs to be escaped, based on the previous rules. Starting in Rudder 7.0, values in Technique Editor are escaped properly.
```

File: src/reference/modules/usage/pages/configuration_management.adoc [hunk 4]
```diff
@@ -99,7 +99,23 @@ if you feel the generated policies should be modified (for instance, if you chan
...
+
+Prior to upgrading from 6.X to 7.0, you need to review values in Techniques from the Technique Editor to unescape ", \ and `
```
Comment (amousset) [inline src/reference/modules/usage/pages/configuration_management.adoc:115]: this is risky as it may apply broken policies on the systems

**Pull request merged**